src/App.js

```diff
@@ -1,13 +1,23 @@
 import './App.css';
 import React, { Component } from 'react';
+import {
+  Routes,
+  Route,
+} from 'react-router-dom';
 import Calculate from './components/Calculator';
+import Home from './components/Home';
+import Quotes from './components/Quotes';
 
 // eslint-disable-next-line react/prefer-stateless-function
 export class App extends Component {
   render() {
     return (
       <div className="App">
-        <Calculate />
+        <Routes>
+          <Route path="/home" element={<Home />} />
```

Comment:
- [ ] Good job so far 👍 Kindly set the path for the home element to be `path="/"` which is the default path because right now when I start your server I'm taken to a blank page which should not be the case.